README.md

```diff
@@ -70,6 +70,11 @@ Now you will see a review with linting errors...
     $ terrascan scan -d path/to/terraform/file -o json |lintly --format=terrascan
     ```
 
+-  [trivy](https://github.com/aquasecurity/trivy)
+    ```
+    $ trivy --quiet fs -f json path/to/directory/ |lintly --format=trivy
```

Comment:
Mind fixing this and the two above here? It's a nitpick but it'd be more readable if you add a space between the pipe and `lintly` (i.e., `... path/to/directory/ | lintly ...`)

Reply:
ha! This has been getting on my OCD too!